src/pset/map/global.rs

```diff
@@ -80,55 +108,40 @@ pub struct Global {
     /// Global transaction data
     #[cfg_attr(feature = "serde", serde(flatten))]
     pub tx_data: TxData,
-    /// The version number of this PSET. If omitted, the version number is 0.
+    /// The version number of this PSET. Must be present.
     pub version: u32,
     /// A global map from extended public keys to the used key fingerprint and
     /// derivation path as defined by BIP 32
     pub xpub: BTreeMap<ExtendedPubKey, KeySource>,
     /// Global proprietary key-value pairs.
-    #[cfg_attr(feature = "serde", serde(with = "::serde_utils::btreemap_as_seq_byte_values"))]
-    pub proprietary: BTreeMap<raw::ProprietaryKey, Vec<u8>>,
+    #[cfg_attr(feature = "serde", serde(flatten))]
+    pub proprietary: GlobalProp,
```

Comment:
Not entirely sure if I should be adding these fields to the proprietary field or if I should just add these as normal fields.